### DISCUSSION TITLE
Issue with Icons used for filtering.

### DISCUSSION THREAD

**Discussion by atulhajare** [Budibase/budibase#9164] [Help]
Hi Budibase team,
Was working on a project using budibase. After the recent update (version 2.2.9 ) have been facing issues with filtering.

Earlier was able to achieve filtering when I jump from screen 1 to screen 2 using icons.
I had two tables and created one to many relations between them.
For example, when I select a row from table 1 (on screen 1) and clicked the icon. 
It used to navigate me to screen 2 and was able to show the value selected from the row in a text field.
Using the text field I was filtering the table on screen 2.

Now, I am able to show the default value in the screen 2 text field. But it does not filter the record in table 2.
I have provided a demo application to reproduce the issue.
Table one:  Inventory
Table two:  Vendor

Just have a look into the demo app and you will get the overall idea of the issue.

[Demo-export-1672053520155.tar.gz](https://github.com/Budibase/budibase/files/10303218/Demo-export-1672053520155.tar.gz)

**Comment by melohagan** [comment]
Hey @atulhajare 

I think this is the same issue here: https://github.com/Budibase/budibase/issues/9104

I suggested a workaround in the meantime at the bottom of the thread.
